src/hooks/use-get-params.ts

```diff
@@ -1,12 +1,15 @@
 import { useSearchParams } from 'react-router-dom';
+import SearchParam from '../types/search-param';
 
 export default function useGetParams() {
   const [searchParams, setSearchParams] = useSearchParams();
 
-  const search = searchParams.get('search');
-  const sortBy = searchParams.get('sortBy') ?? 'release_date';
-  const filter = searchParams.get('filter') ?? 'all';
-  const offset = Number(searchParams.get('offset'));
+  const { Search, SortBy, Filter, Offset } = SearchParam;
+
+  const search = searchParams.get(Search);
+  const sortBy = searchParams.get(SortBy) ?? 'release_date';
+  const filter = searchParams.get(Filter) ?? 'all';
+  const offset = Number(searchParams.get(Offset));
 
   return { search, sortBy, filter, offset, setSearchParams };
```

Comment:
It would be better to have something like this
```
{
   [SearchParam.Search]: searchParams.get(SearchParam.Search);
   [SearchParam.SortOrder]: searchParams.get(SearchParam.SortOrder) ?? 'release_date';
   ...
}

and when you use 

const { [SearchParam.Search]: search } = useGetParams();

it will be easier to support, rename and maintain
```